lib/cdo/script_constants.rb

```diff
@@ -74,10 +74,17 @@ module ScriptConstants
     ],
     hoc: [
       # Note that now multiple scripts can be an 'hour of code' script.
-      # If adding a script here,
-      # you must also update the cdo-tutorials gsheet so the end of script API works
+      # If adding a script here, you must also update the cdo-tutorials gsheet
+      # so the end of script API works; specifically, there needs to be a row
+      # with code_s matching the script name (in quotes) in this list.
 
       nil,
+      POEM_ART = 'poem-art'.freeze, # 2021 hour of code
```

Comment:
These names are planning on being updated to append `-2021` right? Any reason to make this change before those names are updated?

Reply:
I just learned about that and am happy to hold off until they are ready.

Reply:
Hey @breville - It looks like I will be making those copies After 12pm PT on 11/11. I will keep you updated as I make the updates. I could also just take this off your hands if you want and include it as part of that work?

Reply:
@dmcavoy If you'd like to take over this PR, that would be great.  We don't want these tutorials appearing in the course assignment dropdown until next week; I'm still curious when this PR can be merged to ensure that.